lib/susedistribution.pm

```diff
@@ -174,7 +175,7 @@ sub x11_start_program {
     $args{valid}         //= 1;
     $args{target_match}  //= $program;
     $args{match_no_wait} //= 0;
-    die "no desktop-runner available on minimalx" if check_var('DESKTOP', 'minimalx');
```

Comment:
can you explain why you remove this ? minimalx desktop get some desktop runner ?

Reply:
yes, icewm has a runner, see the white textfield in https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/702/diffs#1b0fe96bd6d1f774d20df892ed6082f77b895328 or triggered by 'super-spc'. Also see the last shortcut mentioned in http://www.icewm.org/manual/icewm-6.html . Actually I think many people assumed icewm would have no desktop runner and IIRC we have even some tests disabled for this reason. Maybe this was not always like this?